OrbitCore/FramePointerValidator.h

```diff
@@ -0,0 +1,15 @@
+#ifndef ORBIT_CORE_FRAME_POINTER_VALIDATOR_H_
+#define ORBIT_CORE_FRAME_POINTER_VALIDATOR_H_
+
+#include <vector>
+
+#include "OrbitFunction.h"
+
+class FramePointerValidator {
+ public:
+  static std::vector<std::shared_ptr<Function>> GetFpoFunctions(
```

Comment:
Please add a unit-test

Reply:
I added a test. This is currently failing, as function sizes seem to be calculated incorrectly.